include/net/net_pkt.h

```diff
@@ -594,31 +594,31 @@ static inline u8_t *net_pkt_ll(struct net_pkt *pkt)
 	return net_pkt_ip_data(pkt) - net_pkt_ll_reserve(pkt);
 }
 
-static inline struct net_linkaddr *net_pkt_ll_src(struct net_pkt *pkt)
+static inline struct net_linkaddr *net_pkt_lladdr_src(struct net_pkt *pkt)
 {
 	return &pkt->lladdr_src;
 }
 
-static inline struct net_linkaddr *net_pkt_ll_dst(struct net_pkt *pkt)
+static inline struct net_linkaddr *net_pkt_lladdr_dst(struct net_pkt *pkt)
 {
 	return &pkt->lladdr_dst;
 }
 
-static inline void net_pkt_ll_clear(struct net_pkt *pkt)
-{
-	memset(net_pkt_ll(pkt), 0, net_pkt_ll_reserve(pkt));
-	net_pkt_ll_src(pkt)->addr = NULL;
-	net_pkt_ll_src(pkt)->len = 0;
-}
-
-static inline void net_pkt_ll_swap(struct net_pkt *pkt)
+static inline void net_pkt_lladdr_swap(struct net_pkt *pkt)
 {
 	u8_t *addr = net_pkt_ll_src(pkt)->addr;
 
 	net_pkt_ll_src(pkt)->addr = net_pkt_ll_dst(pkt)->addr;
 	net_pkt_ll_dst(pkt)->addr = addr;
 }
 
+static inline void net_pkt_ll_clear(struct net_pkt *pkt)
+{
+	memset(net_pkt_ll(pkt), 0, net_pkt_ll_reserve(pkt));
+	net_pkt_ll_src(pkt)->addr = NULL;
```

Comment:
Shouldn't this be `net_pkt_lladdr_src()`?

Reply:
yes, grepped everywhere but in include... ^^